openapi/Swarm.yaml

```diff
@@ -723,6 +727,13 @@ paths:
           description: Default response
 
   "/stamps/{id}":
+    parameters:
+      - in: path
+        name: id
+        schema:
+          $ref: "SwarmCommon.yaml#/components/schemas/SwarmAddress"
+        required: true
+        description: Swarm address of the stamp
```

Comment:
actually the batchID is not an address, it is just a uuid we get from the chain AFAIK. not sure exactly how it is calculated. @ralph-pichler ?

Reply:
it is a 32 byte hash based on the creator address and random nonce

Reply:
so maybe it would be good to have this as a 32 byte hex string (I think we have this sort type in the OpenAPI spec) :+1:

Reply:
`SwarmAddress` is just that ;-)

Reply:
Created separate type for that